source/include/cfg/cs_SerialConfig.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
This file isn't a config, it's just typedefs and enums really.
I think it could be moved to `protocol/cs_SerialTypes.h` or so.